modules/pixi/styles.js

```diff
@@ -37,42 +41,55 @@ export const STYLES = {
     stroke: { width: 3, color: 0xcccccc, alpha: 1, cap: PIXI.LINE_CAP.ROUND, join: PIXI.LINE_JOIN.ROUND }
   },
 
-  red: {
-    fill: { color: 0xe06e5f, alpha: 0.3 }   // rgb(224, 110, 95)
-  },
-  green: {
-    fill: { color: 0x8cd05f, alpha: 0.3 }   // rgb(140, 208, 95)
-  },
-  blue: {
-    fill: { color: 0x77d4de, alpha: 0.3 }   // rgb(119, 211, 222)
-  },
-  yellow: {
-    fill: { color: 0xffff94, alpha: 0.25 }  // rgb(255, 255, 148)
-  },
-  gold: {
-    fill: { color: 0xc4be19, alpha: 0.3 }   // rgb(196, 189, 25)
-  },
-  orange: {
-    fill: { color: 0xd6881a, alpha: 0.3 }   // rgb(214, 136, 26)
-  },
-  pink: {
-    fill: { color: 0xe3a4f5, alpha: 0.3 }   // rgb(228, 164, 245)
-  },
-  teal: {
-    fill: { color: 0x99e1aa, alpha: 0.3 }   // rgb(153, 225, 170)
-  },
-  lightgreen: {
-    fill: { color: 0xbee83f, alpha: 0.3 }   // rgb(191, 232, 63)
-  },
-  tan: {
-    fill: { color: 0xf5dcba, alpha: 0.3 }   // rgb(245, 220, 186)
-  },
-  darkgray: {
-    fill: { color: 0x8c8c8c, alpha: 0.5 }   // rgb(140, 140, 140)
-  },
-  lightgray: {
-    fill: { color: 0xaaaaaa, alpha: 0.3 }   // rgb(170, 170, 170)
-  },
```

Comment:
@Bonkles the lines above are the original code, and below is where I started to refactor. Is this on the right track for what you had in mind?

Reply:
Yes indeedly-do! We're now starting to separate concerns. We want Colors is in charge of... well, colors, and styles just farms all its color concerns to the ColorSystem.